dev/mocks.js

```diff
@@ -144,6 +143,36 @@ const passToProvider = (url, request) => {
   throw Error('Unexpected Request')
 }
 
+export const sendResponse = ({ body, options, res, root, host }) => {
+  res.statusCode = options.status
+  const entries = Object.entries(options.headers || {})
+  for (const [k, v] of entries) res.setHeader(k, v)
+
+  // Default case, just return the body
+  if (options.status !== 301) return res.end(body)
+
+  // Make cookies insecure for http support
+  if (options.headers['set-cookie']) {
+    const cookie = options.headers['set-cookie'].split('; ', 3).join('; ')
+    res.setHeader('set-cookie', cookie)
+  }
+
+  // If it's a local redirection, we stop here
+  if (options.headers.location[0] === '/') {
+    res.setHeader('location', `${host || ''}${options.headers.location}`)
```

Comment:
`host` must be optionally set on the CI server so redirections use the correct host